pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -20,6 +20,7 @@
 , javacSupport ? false, javacPackages ? [ openjdk8 ]
 , odbcSupport ? false, odbcPackages ? [ unixODBC ]
 , withSystemd ? stdenv.isLinux # systemd support in epmd
+, opensslPackage ? openssl
```

Comment:
Somehow it feels weird still that this is called `opensslPackage` instead of just `openssl`.